sympy/solvers/solveset.py

```diff
@@ -234,6 +231,69 @@ def _invert_complex(f, g_ys, symbol):
     return (f, g_ys)
 
 
+def _invert_abs(f, g_ys, symbol):
+    """Helper function for inverting absolute value functions"""
```

Comment:
I needed a review of this as I was not sure about what should go in the docs.
Thanks.